### PR TITLE
English README readability 

### PR DESCRIPTION
**Pull request by Ma11ock**
The README has spelling and grammar mistakes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -250,41 +250,41 @@ We have split this task in some steps:
...
+  The `intel_startCPU()` function has the purpose of enabling the cpu indicated by parameter, calling to `startup_cpu()` to raise the Startup IPI, and check if the cpu has been enabled correctly.
   
-  To write this function, we have based in [XNU's `intel_startCPU()` function](https://github.com/nneonneo/osx-10.9-opensource/blob/f5a0b24e4d98574462c8f5e5dfcf0d37ef7e0764/xnu-2422.1.72/osfmk/i386/mp.c#L423), replacing its calls to the *gnumach* equivalent, and removing garbage code blocks.
+  This function is based on [XNU's `intel_startCPU()` function](https://github.com/nneonneo/osx-10.9-opensource/blob/f5a0b24e4d98574462c8f5e5dfcf0d37ef7e0764/xnu-2422.1.72/osfmk/i386/mp.c#L423), replacing its calls to the *gnumach* equivalent, and removing garbage code blocks.
   
 - **Raise Startup IPI and initialize cpu**
 
-  *gnumach* doesn't include any function to raise the Startup IPI, so we have implemented this functions based in Min_SMP [`cpu.c`](https://github.com/AlmuHS/Min_SMP/blob/master/cpu.c) and [`mp.c`](https://github.com/AlmuHS/Min_SMP/blob/master/mp.c )functions:
```
Comment (AlmuHS) [inline README.md:270]: The fix is not exact. My work in gnumach is derived from the work previously done in MinSMP

File: README.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ More info in: <https://www.gnu.org/software/hurd/microkernel/mach/gnumach/buildi
 - In the [Min_SMP](https://github.com/AlmuHS/Min_SMP/) test environment, the cpus are detected and started correctly
 	+ I need to implement APIC configuration
 - In *gnumach*, the number of cpus and its lapic structures are detected and enumerated correctly
-- ioapic enumeration fails to work correctly
+- ioapic enumeration fails
```
Comment (AlmuHS) [inline README.md]: ioapic enumeration not fails, it works fine

File: README.md [hunk 3]
```diff
@@ -133,7 +133,7 @@ More info in: <https://www.gnu.org/software/hurd/microkernel/mach/gnumach/buildi
 - In the [Min_SMP](https://github.com/AlmuHS/Min_SMP/) test environment, the cpus are detected and started correctly
 	+ I need to implement APIC configuration
 - In *gnumach*, the number of cpus and its lapic structures are detected and enumerated correctly
-- ioapic enumeration fails to work correctly
```
Comment (AlmuHS) [inline README.md]: ioapic enumeration seems to work correctly

**Review by AlmuHS — COMMENTED** [review]
Before accept the PR, I need that you fix some mistakes.

**Comment by Ma11ock** [comment]
The latest commit `587b871' fixes the inaccuracies plus a few more grammar hiccups.

**Review by AlmuHS — APPROVED** [review]

**Pull request merged**